<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainHomePage</class>
 <widget class="QWidget" name="MainHomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>431</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainHomePage{
border-image: url(&quot;F:/tmp/MESApp1.1/res/main_bk.png&quot;) 50 00 0 0 repeat;
}

QGroupBox
{
background-color: rgba(85, 85, 127, 50);
border-radius: 10px;
border: 1px solid rgb(106, 163, 193);
}
#widget_back
{
background-color: rgba(85, 85, 127, 50);
border-radius: 10px;
border: 1px solid rgb(106, 163, 193);
}
#widget_back_3
{
background-color: rgba(85, 85, 127, 50);
border-radius: 10px;
border: 1px solid rgb(106, 163, 193);
}
#btItemNext {font: bold  12px;
	color: rgb(255, 255, 127);
border-bottom-left-radius:20px;
border-bottom-right-radius:20px;
border: 1px solid rgba(255, 255, 255,100);
background-color: rgba(85, 170, 127,100);}

#btItemUp{
font: bold  12px;
	color: rgb(255, 255, 127);
border-top-left-radius:20px;
border-top-right-radius:20px;
border: 1px solid rgba(255, 255, 255,100);
background-color: rgba(85, 170, 127,100);}

#groupBox1 QLabel {
color: rgb(255, 255, 255);
font: bold 18px;
background-color: transparent;
}
#labelSumTitle {
color: rgb(255, 255, 255);
font: bold 24px;
padding-left:2px;
}
#labelClassTitle {
color: rgb(255, 255, 255);
font: bold 24px;
padding-left:2px;
}

#MainHomePage QTableWidget {
	background-color: transparent;
	font: bold 24px;	
	color: rgb(234, 234, 116);
}
#MainHomePage QHeaderView{  
	background-color: transparent;
}
#MainHomePage QHeaderView::section {	
	background-color: transparent;
	color: rgb(255, 255, 255);
	font: bold 16px;
	border: 0px solid #6c6c6c;
	height: 40px;
}
#MainHomePage QTableWidget::item {
	background-color: rgba(130, 157, 255, 20);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>541</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_order">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>派工单号 -</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_num">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>产品编号 -</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_name">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>产品名称 -</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btItemUp">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>5</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>上件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btItemNext">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>65</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>下件</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>541</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelSumTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>总
计</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetCount">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>99</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>51</number>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>51</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>99</number>
    </attribute>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>派工数</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>模次数</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>良品数</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>生产数</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>次品数</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>-</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>-</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>-</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>-</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>-</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>541</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelClassTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>本
班</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetClass">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>124</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>124</number>
    </attribute>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>生产数</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>良品数</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>次品数</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>次品率</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>-</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>-</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>-</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>-</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_back" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>8</y>
     <width>221</width>
     <height>205</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_back_3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="runStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 90px;
background-color: rgb(0, 172, 0);
background-color: red;
background-color:green;
font:bold 50px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>运</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
